gensim/models/onlineatvb.py

```diff
@@ -371,6 +389,8 @@ def word_bound(self, Elogtheta, Elogbeta, doc_ids=None):
 
         # NOTE: computing the bound this way is very numerically unstable, which is why
         # "logsumexp" is used in the LDA code.
+        # NOTE: computing bound is very very computationally intensive. I could, for example,
```

Comment:
Who is "I"? If it's a note to self, please mark as `FIXME` (and make sure to address all FIXMEs before merging).